### ISSUE TITLE
React Hot Loader 3 support needed

### ISSUE THREAD

**Issue by BerndWessels** [ezekielchentnik/preact-pwa#8]
Hi

This is an awesome project, but without Hot Module Reloading the development experience is terrible.

Can you add support for [React Hot Loader 3](https://github.com/gaearon/react-hot-boilerplate/pull/61) please?

Thank you

**Comment by ezekielchentnik** [comment]
@BerndWessels I agree, this is something I want; that, and general speed up of build time.  I'll give it a try.

Any contribs related to this topic are also welcome :)  Cheers, thanks for the input.

**Comment by BerndWessels** [comment]
Thanks, that's awesome that you are considering it. I'd love to contribute, but have only experience with webpack, so might need to do some catching up on rollup before I can be of use ;)
